_posts/2025-08-27-offer-multiple-ways-to-contact-you.md

```diff
@@ -0,0 +1,125 @@
+
+So, what are the ways to communicate? And what are the pros and cons?
+
+## Different ways to get in touch
+
+### Social media and instant messaging
+
+Services like WhatsApp or X (formerly Twitter) can be useful, it’s widely used, low-barrier, and great for quick questions.
+
+But don’t assume everyone uses it. Many of people are moving away from WhatsApp due to Meta’s privacy issues. Some prefer Signal or don’t want to share personal stuff over WhatsApp at all.
+
+So keep in mind: not everyone uses what you prefer.
+
+### Phone call
+
+Many people prefer a phone call, real person, instant answers. Definitely offer it as an option if possible.
+
+User research shows that people with low literacy prefer calling first:
+
+> “I always try calling first. If that doesn’t work, I’ll check the website to find another way.”
+
+But for some, like deaf, hard-of-hearing, shy, autistic, or stuttering people, phone calls can be tough. The same goes for people who struggle with your native language. Image you're in a noisy train and don’t want to talk about private stuff.
+
+Some callers even get hung up on if they take too long to respond. Wouter Honselaar, who stutters, said in the interview [I am stronger than stuttering](https://www.a11y-collective.com/blog/interview-with-wouter-i-am-stronger-than-stuttering/):
+
+> “Once, someone ended the call because they thought the line had gone dead.”
+
+Also, phone calls often only work during office hours. If you work during those hours and can’t make personal calls, it’s not an option.
+
+I asked my deaf followers on LinkedIn:
+
+> “What do you do when the only way to contact a website is by phone, and you *really* need to get in touch?”
+
+[Marie Van Driessche](https://www.linkedin.com/in/marie-van-driessche/) replied:
+
+> “I use a [video relay service](https://en.wikipedia.org/wiki/Video_relay_service) via the Tolkcontact app. So I get a translator to call them for me. It works, but I have to arrange it all myself. It’d be nice if email or WhatsApp were options.”
+
+[Meryl Evans](https://www.linkedin.com/in/meryl/) added:
+
+> “Offer text-based options like email, SMS, live chat with a real person, or a video relay. Let people choose what contact details they share. Don’t make phone numbers required.”
+
+### Email
+
+The nice thing about email is that you can send one whenever you want.
+
+But not everyone finds it easy to write an email, especially if they’re not confident in your langauge, have dyslexia or low literacy. Some people may need help just to write a message.
+
+### Contact Form
+
+Contact forms are handy, you ask exactly what you need, and users can fill it out in an organized way.
+
+But forms can be a real barrier for people who aren’t great with tech. Especially if the form is long or complicated. Always test your forms with real users, including people with disabilities.
```

Comment:
"But forms can be a real barrier for people who aren’t great with tech, especially if the form is long or complicated."